examples/Demos/caduceus.scn

```diff
@@ -7,7 +7,7 @@
     <CollisionPipeline depth="15" verbose="0" draw="0" />
     <BruteForceDetection name="N2" />
     <MinProximityIntersection name="Proximity" alarmDistance="1.5" contactDistance="1" />
-
+    <InteractiveCamera position="0 30 90" lookAt="0 30 0"/>
```

Comment:
I added an interactive camera for the caduceus example. By adding it, it's now possible to do some footage of the snake with HeadlessRecorder. This plugin need an interactive camera to place correctly the camera.